packages/components/molecules/f-searchbox/src/store/searchbox.module.js

```diff
@@ -174,6 +185,27 @@ export default {
 
         [SET_AUTO_COMPLETE_AVAILABILITY]: (state, isAutocompleteEnabled) => {
             state.isAutocompleteEnabled = isAutocompleteEnabled;
+        },
+
+        /**
+         * Map API response items and filter on `Postcode` value. We don't want to
+         * return `Address` types here only postcode results for users.
+         *
+         * @param state
+         * @param suggestions
+         */
+        [SET_PARTIAL_ADDRESS_SUGGESTIONS]: (state, suggestions) => {
+            state.suggestions = suggestions.Items.map(({
+                Description,
+                Text,
+                Id,
+                Type
+            }) => ({
+                Description,
```

Comment:
You _could` map these into camel case? Or would that cause issues elsewhere?

Reply:
WIll take a look in the next PR if that's ok. I think i did try that in the old searchbox but it was a while back so can't remember if it worked ok or not :D